src/it/resources/org/checkstyle/suppressionxpathfilter/packagename/SuppressionXpathRegressionPackageNameTwo.java

```diff
@@ -0,0 +1,4 @@
+package org.checkstyle.suppressionxpathfilter.packagename; // warn
```

Comment:
Same as in Two.
Please make package name wrapped in few lines, to make it a bit different.
And please add comment before `package`

Reply:
done